packages/react-celo/src/constants.tsx

```diff
@@ -248,16 +248,16 @@ export const PROVIDERS: {
 };
 
 export const images = {
```

Comment:
My vs code find all doesnt show any usages of this images object. plus it duplicates the icon field above. I think we can remove